src/components/editor/editor-pane/tool-bar/upload-file-picker.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * SPDX-FileCopyrightText: 2020 The HedgeDoc developers (see AUTHORS file)
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only
+ */
+
+import { Editor } from 'codemirror'
+import React from 'react'
+import { handleUpload } from '../upload-handler'
+
+export const supportedMimeTypes: string[] = [
+  'application/pdf',
+  'image/apng',
+  'image/bmp',
+  'image/gif',
+  'image/heif',
+  'image/heic',
+  'image/heif-sequence',
+  'image/heic-sequence',
+  'image/jpeg, image/png',
```

Comment:
Any reason for having both of these in the same line?

Reply:
no this was a typo. fixed now